src/autolabel/models/google.py

```diff
@@ -34,13 +35,14 @@ def __init__(
         self,
         config: AutolabelConfig,
         cache: BaseCache = None,
+        tokenizer: Optional[AutoTokenizer] = None,
```

Comment:
these models dont use the tokenizer at all

Reply:
No, they don't. But we need as its part of the interface